### ISSUE TITLE
Eslint plugin doesn't allow display: "-webkit-box", WebkitLineClamp, and WebkitBoxOrient

### ISSUE THREAD

**Issue by denexapp** [facebook/stylex#404]
### Describe the issue

Hi, first of all, thank you for open sourcing this library.

There's a technic to limit amount of lines in a text block using the following set of prefixed properties ([mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp)), which are supported by all browsers: 
edit: updated the example
```

const styles = stylex.create({
  element: {
    display: "-webkit-box",
    WebkitLineClamp: 2,
    WebkitBoxOrient: "vertical"
  }
})
```
The eslint plugin doesn't like all of these three lines, although there's no alternative way to implement this, at least to my knowledge.

### Expected behavior

The eslint plugin should not mark these properties as not allowed or unknown.

### Steps to reproduce

Add @stylexjs/eslint-plugin: 0.4.1 to the project, then create a stylex style with the following lines: 
```
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-line-clamp: 3;
```

### Additional comments

Feels related to https://github.com/facebook/stylex/issues/390

**Comment by yousoumar** [comment]
Hi @denexapp!  The specified version of the ESLint plugin support these properties. Have you done it this way, for example:

```
const styles = stylex.create({
  div: {
    display: "-webkit-box",
    WebkitBoxOrient: "vertical",
    WebkitLineClamp: 3,
  },
});
```

**Comment by yousoumar** [comment]
To be sure, I also checked the plugin code and here are the rules for the properties listed:

```js
const display = makeUnionRule(
  makeLiteralRule('none'),
  makeLiteralRule('inline'),
  makeLiteralRule('block'),
  makeLiteralRule('list-item'),
  makeLiteralRule('inline-list-item'),
  makeLiteralRule('inline-block'),
  makeLiteralRule('inline-table'),
  makeLiteralRule('table'),
  makeLiteralRule('table-cell'),
  makeLiteralRule('table-column'),
  makeLiteralRule('table-column-group'),
  makeLiteralRule('table-footer-group'),
  makeLiteralRule('table-header-group'),
  makeLiteralRule('table-row'),
  makeLiteralRule('table-row-group'),
  makeLiteralRule('flex'),
  makeLiteralRule('inline-flex'),
  makeLiteralRule('grid'),
  makeLiteralRule('inline-grid'),
  makeLiteralRule('-webkit-box'),
  makeLiteralRule('run-in'),
  makeLiteralRule('ruby'),
  makeLiteralRule('ruby-base'),
  makeLiteralRule('ruby-text'),
  makeLiteralRule('ruby-base-container'),
  makeLiteralRule('ruby-text-container'),
  makeLiteralRule('contents'),
);
```
```js
  WebkitBoxOrient: makeUnionRule(
    'horizontal',
    'vertical',
    'inline-axis',
    'block-axis',
  ),
  WebkitLineClamp: isNumber,
```

**Comment by yousoumar** [comment]
Hi @denexapp! The latest release of the plugin support these properties. Upgrade yours with:

```
npm i -D @stylexjs/eslint-plugin@latest
``` 
And restart your IDE (I had to for Visual Studio Code).

**Comment by necolas** [comment]
I've always wanted us to polyfill the draft property `lineClamp`. It's meant to provide the same functionality (without changing the display) and wouldn't require these old properties to be allowed. We'd have a compiler option for disabling individual polyfills. That's also why I like the idea of stylex internalising vendor prefixing while it's still needed

**Comment by nmn** [comment]
This issue has been fixed on `main` and will be out in the next release. In the meantime, you can suppress the lint error and continue to use it.

**Comment by yousoumar** [comment]
I'm a little confused here. @nmn you are saying it's not released and yet on the version 0.4.1 I have the same issue as @denexapp but not on the latest `0.5.1` version.

**Comment by nmn** [comment]
@yousoumar You're right. The fix is already out in `0.5.1`. I wasn't sure it was out as it was merged just before release.

**Comment by denexapp** [comment]
Yeah, I confirm it works on the latest version.

Apparently, if you use the `npm update` command for a package below 1.0.0 version, it won't update to the next minor version.
